tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -59,324 +63,321 @@ create_boundary_condition() {
       Direction<3>::upper_zeta(), 50);
 }
 
-auto make_domain_creator(const std::string& opt_string,
-                         const bool use_boundary_conditions) {
-  if (use_boundary_conditions) {
-    return TestHelpers::test_option_tag<
-        domain::OptionTags::DomainCreator<3>,
-        TestHelpers::domain::BoundaryConditions::
-            MetavariablesWithBoundaryConditions<3, domain::creators::Sphere>>(
-        opt_string + std::string{"  BoundaryCondition:\n"
-                                 "    TestBoundaryCondition:\n"
-                                 "      Direction: upper-zeta\n"
-                                 "      BlockId: 50\n"});
+// Calculate block logical coordinates
```

Comment:
Could you explain a bit more what sort of block logical coords you're generating here?